.github/workflows/finishReleaseCycle.yml

```diff
@@ -50,19 +48,44 @@ jobs:
             This issue either has unchecked items or has not yet been marked with the `:shipit:` emoji of approval.
             Reopening!
 
+      - if: ${{ failure() }}
+        uses: Expensify/App/.github/actions/composite/announceFailedWorkflowInSlack@main
+        with:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+
   # Update the production branch to trigger the production deploy.
   updateProduction:
     runs-on: ubuntu-latest
     needs: validate
     if: ${{ fromJSON(needs.validate.outputs.isValid) }}
     steps:
-      - name: Update production branch
-        uses: Expensify/App/.github/actions/composite/updateProtectedBranch@main
+      - uses: actions/checkout@v3
+        with:
+          ref: staging
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - uses: Expensify/App/.github/actions/composite/setupGitForOSBotify@main
         with:
-          TARGET_BRANCH: production
-          OS_BOTIFY_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
+      - name: Update production branch
+        run: |
+          # Get the production branch locally
+          git switch production
+
+          # Delete the production branch and re-create it from staging
+          git switch staging
+          git branch -D production
+          git switch -c production
+
+          # Force-update push the new production branch.
+          git push --force origin production
```

Comment:
Can we delete the remote `production` and not `--force` it?

Reply:
I don't think that it's possible to delete and re-created a branch without `--force`?

Reply:
In any case, AFAIK this is the same as what we do in web and I rather do the same than try out a new way

Reply:
Sounds good 👍